de.metas.business/src/main/java/org/adempiere/impexp/bpartner/BPartnerImportProcess.java

```diff
@@ -245,7 +246,11 @@ private final void createBPPrintFormatIfNeeded(@NonNull final I_I_BPartner impor
 					.adOrgId(importRecord.getAD_Org_ID())
 					.build());
 
-			final int AD_PrintFormat_ID =  Services.get(ISysConfigBL.class).getIntValue(BPARTNER_IMPORTPROCESS_BPPrintFormat, 1000015);
+			final int AD_PrintFormat_ID =  Services.get(ISysConfigBL.class).getIntValue(BPARTNER_IMPORTPROCESS_BPPrintFormatId, -1);
+			if (AD_PrintFormat_ID <= 0 )
+			{
+				throw new AdempiereException(BPARTNER_IMPORTPROCESS_BPPrintFormatId_ErrorMsg);
```

Comment:
your message won't be translated.
you need to:
* use `@`, before and after
* or use org.adempiere.exceptions.AdempiereException.AdempiereException(String, Object[])
* or use org.adempiere.exceptions.AdempiereException.AdempiereException(ITranslatableString) and de.metas.i18n.IMsgBL.getTranslatableMsgText(String, Object...)

Reply:
Thx! Fixed! I totally forgot about @@

Reply:
or better use AdempiereException.ofADMessage (I just introduced it in latest master)